templates/web/base/admin/bodies/contact-form.html

```diff
@@ -40,6 +40,11 @@
             <label for="state-deleted">[% loc('Deleted') %]</label>
             <p class="form-hint" id="state-deleted-hint">[% loc('Prevent new reports from using this category, <em>and</em> also remove it from map filters.') %]</p>
         </div>
+        <div class="form-check form-check--inline">
+            <input type="radio" name="state" id="state-deleted" aria-describedby="state-deleted-hint" value="staff"[% ' checked' IF contact.state == 'staff' %]>
+            <label for="state-deleted">[% loc('Staff only') %]</label>
+            <p class="form-hint" id="state-deleted-hint">[% loc('Only staff users will be able to add reports in this category.') %]</p>
```

Comment:
Looks like you've accidentally reused `deleted` on these three lines instead of e.g. `staff`.

Also - should it be made clear that reports will be publicly visible unless the "Hide reports made in this category" box is also ticked? Would it be going too far to tick that box automatically (or even make it mandatory) when the 'staff' option is selected?

Reply:
Hmm, I realise it's been asked for here in terms of making private reports, but I could see there being categories that were staff only that could nonetheless be normal public reports, so I think I'll leave it for now (sure you'll be right and they'll not do it right).